civismlext/stacking.py

```diff
@@ -96,11 +96,13 @@ def get_params(self, deep=True):
             return out
         # If deep, extract parameters from estimators too
         est_list = getattr(self, 'estimator_list')
```

Comment:
Do we need something like `getattr(self, 'estimator_list', [])`? Like can this be set to `None`?

Reply:
`estimator_list` is a required argument, and it has to be a list, so that shouldn't be an issue. It throws an error if you try to pass `None`.

Reply:
Sweet!